src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -109,6 +110,7 @@ export default function ConfirmedAppointmentDetailsPage() {
           useV2={useV2}
         />
       )}
+      {isCommunityCare && <DetailsCC appointment={appointment} useV2={false} />}
```

Comment:
set the `useV2` parameter to the variable otherwise the argument will always be false
```suggestion
      {isCommunityCare && <DetailsCC appointment={appointment} useV2={useV2} />}
```

Reply:
I did add this on my end before I saw your comment.